src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py

```diff
@@ -101,6 +102,33 @@ def boost_low_volume_projects(context: TaskContext) -> None:
             boost_low_volume_projects_of_org.delay(org_id, projects_with_tx_count_and_rates)
 
 
+@instrumented_task(
+    name="sentry.dynamic_sampling.boost_low_volume_projects_of_org_with_query",
+    queue="dynamicsampling",
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=25 * 60,
+    time_limit=2 * 60 + 5,
+    silo_mode=SiloMode.REGION,
+)
+@dynamic_sampling_task
+def boost_low_volume_projects_of_org_with_query(
```

Comment:
Since this is very similar to `boost_low_volume_projects_of_org`, could you add docstrings to the two tasks that shows their differences and when which is to be used?

Reply:
Done with [04b5ec2](https://github.com/getsentry/sentry/pull/79582/commits/04b5ec29243ce75d3a156be6438b033e67016cc1)